### ISSUE TITLE
5.2.0 requires peer dependencies firebase-tools, fuzzy, inquirer, and inquirer-autocomplete-prompt

### ISSUE THREAD

**Issue by katowulf** [angular/angularfire2#2094]
Installing 5.2.0 lists new peer dependencies. Am I required to install firebase-tools locally (as apposed to `-g`) as well as fuzzy, inquirer, and inquirer-autocomplete-prompt? 

If so, maybe docs update. If not, then Kato.insertConfusion().

### Version info

Package.json
```
 "dependencies": {
    "@angular/animations": "^7.1.4",
    "@angular/cdk": "^7.1.1",
    "@angular/common": "^7.1.4",
    "@angular/compiler": "^7.1.4",
    "@angular/core": "^7.1.4",
    "@angular/fire": "^5.2.0",
    "@angular/forms": "^7.1.4",
    "@angular/platform-browser": "^7.1.4",
    "@angular/platform-browser-dynamic": "^7.1.4",
    "@angular/router": "^7.1.4",
    "chart.js": "^2.8.0",
    "core-js": "^2.6.9",
    "firebase": "^5.11.1",
    "font-awesome": "^4.7.0",
    "moment": "^2.24.0",
    "primeflex": "^1.0.0-rc.1",
    "primeicons": "^1.0.0",
    "primeng": "^7.1.3",
    "rxjs": "^6.3.3",
    "tslib": "^1.9.3",
    "zone.js": "^0.8.29"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.11.0",
    "@angular/cli": "^7.1.4",
    "@angular/compiler-cli": "^7.1.4",
    "@angular/language-service": "^7.1.4",
    "@types/jasmine": "~2.8.8",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "~8.9.4",
    "codelyzer": "~4.5.0",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~3.1.1",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~1.1.2",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.11.0",
    "typescript": "~3.1.6"
  }
```

Others:
```
sputnik:public katowulf$ node -v
v8.9.3
sputnik:public katowulf$ npm list -g | grep firebase
├─┬ firebase-tools@6.10.0
│ ├─┬ firebase@2.4.2
```

### How to reproduce these conditions

**Steps to set up and reproduce**

Perform npm-install with the above dependencies.

### Debug output

```
npm WARN @angular/fire@5.2.0 requires a peer of firebase-tools@^6.10.0 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/fire@5.2.0 requires a peer of fuzzy@^0.1.3 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/fire@5.2.0 requires a peer of inquirer@^6.2.2 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/fire@5.2.0 requires a peer of inquirer-autocomplete-prompt@^1.0.1 but none is installed. You must install peer dependencies yourself.
```

### Expected behavior

Less confusion. : )

### Actual behavior

More confusion. : (

**Comment by katowulf** [comment]
I removed node modules and reinstalled for fun, but did not profit.

**Comment by jamesdaniels** [comment]
I should mark them as optional. Used the the schematics.

**Comment by jamesdaniels** [comment]
Fixed in `5.2.1`, thanks for reporting Kato!
